app/views/static_pages/index.html.erb

```diff
@@ -113,7 +142,9 @@
   <% end %>
   <%= render partial: "static_pages/admin" %>
 </main>
-
+<%
```

Comment:
Did you intend to keep this commented out? If so, maybe we remove the code entirely.